addons/point_of_sale/models/pos_order.py

```diff
@@ -334,15 +334,16 @@ def insert_data(data_type, values):
 
     def _reconcile_payments(self):
         for order in self:
-            aml = order.statement_ids.mapped('journal_entry_ids') | order.account_move.line_ids
-            aml = aml.filtered(lambda r: not r.reconciled and r.account_id.internal_type == 'receivable' and r.partner_id == order.partner_id)
+            aml = order.statement_ids.mapped('journal_entry_ids').mapped('line_ids') | order.account_move.line_ids | order.invoice_id.move_id.line_ids
```

Comment:
I was wondering how I could make that mistake, but I realized that `journal_entry_ids` are `account.move.line` from saas-14, but `account.move` in v10.0.